docs/contributing/contributing-rules.md

```diff
@@ -19,13 +26,17 @@ There are two ways in which you can contribute to the Semgrep rules repository:
 
 ### Contributing through Semgrep App (recommended)
 
-To contribute to the Semgrep rules repository through Semgrep App, follow these steps:
+Publish rules in the open-source Semgrep Registry to share them with the Semgrep community. To contribute and publish rules to the Semgrep Registry, follow these steps:
+
 1. Go to [Semgrep App Editor](https://semgrep.dev/orgs/-/editor).
 2. Click **Create New Rule**.
 3. Make one of the following steps:
```

Comment:
"Make"  here isn't correct. "choose"? "Follow"? or simply "one of the following"?

Reply:
Changing to:

`Choose one of the following steps:`